cmd/bridgec/erc20cmd.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/threefoldfoundation/tfchain/cmd/bridgec/internal"
+	"github.com/threefoldfoundation/tfchain/pkg/api"
+	erc20 "github.com/threefoldfoundation/tfchain/pkg/eth/erc20"
+	"github.com/threefoldtech/rivine/pkg/cli"
+)
+
+// createERC20Cmd creates rootcommand for ERC20 and adds a subcommand
+// if rootcommand executed the user will also see the output of the syncing status of ethereum
+func createERC20Cmd(client *internal.CommandLineClient) *cobra.Command {
+	erc20SubCmds := &erc20SubCmds{cli: client}
+
+	// define Rootcommand
+	var (
+		rootCmd = &cobra.Command{
+			Use:   "erc20",
+			Short: "Perform erc20 actions",
+			Run:   erc20SubCmds.getSyncingStatus,
+		}
+		getSyncingStatusCmd = &cobra.Command{
+			Use:   "syncstatus",
+			Short: "Get the ethereum sync status",
```

Comment:
You marked my previous conversation about this line of code as resolved, but only solved it halve, so will copy paste the half that isn't solved yet for convenience:

```suggestion
			Short: "Get the ethereum downloader sync status",
```